src/VisualStudio/VisualBasic/Impl/VSPackage.resx

```diff
@@ -214,4 +214,22 @@ Use enhanced colors;Editor Color Scheme;Inheritance Margin;Import Directives;</v
   <data name="An_empty_Visual_Basic_script_file" xml:space="preserve">
     <value>An empty Visual Basic script file.</value>
   </data>
+  <data name="Always_add_new_line_on_enter" xml:space="preserve">
```

Comment:
Need this because unified settings's enum label can only read from package resources now

Reply:
Can we remove the corresponding strings from their original location, and reference these from the legacy settings page instead?

Reply:
Possible, let me check the LOC time line